games-arcade/ceferino/files/ceferino-0.97.8-fix-audio.patch

```diff
@@ -0,0 +1,11 @@
+--- a/src/audio.h	2013-08-05 16:29:49.000000000 +0200
++++ b/src/audio.h	2021-07-29 18:51:16.668397338 +0200
```

Comment:
Believe already know what to do with these timestamps now :)

Normally be good to have a short description on top too, "fix audio" already says most of it but maybe add something like "12 .wav files for an array with 11 elements invokes undefined behavior."

On as side-note, for simple patch like this you could also use a smaller unified context, using diff's -U 2 or -U 1. Not that I think many devs do this beside me.

Reply:
Updated the patch with `diff -U2` and removed the timestamp :)